### ISSUE TITLE
Play multiple streams at exactly the right time

### ISSUE THREAD

**Issue by JapanYoshi** [goldfire/howler.js#1242]
I have a multi-track background music, and want to fade in/out parts of it depending on the situation. However, in some situations (Because they're rushed ones, I suspect it has to do with loading), one track starts playing before another. How do I resolve this?

**Comment by JapanYoshi** [comment]
I think the issue here is that I tried to play a track before it loads. How do I wait until ALL audios in a group are loaded before playing them?

**Comment by JapanYoshi** [comment]
Is there any update on this?

**Comment by el-wynn** [comment]
Hi, have you been able to reslove this?
Thanks

**Comment by JapanYoshi** [comment]
Not exactly, but you can use the `Howl.state()` method to check if the sound is loaded (returns the string `loaded` if it is), have a function that checks the state of multiple Howls iteratively, and have that function fire as a `Howl.onload`.
